packages/components/carousel/src/use-carousel.ts

```diff
@@ -53,6 +54,27 @@ export const useCarousel = (
   const isCardType = computed(() => props.type === 'card')
   const isVertical = computed(() => props.direction === 'vertical')
 
+  const containerStyle = computed(() => {
+    if (props.height !== 'auto') {
+      return {
+        height: props.height,
+      }
+    }
+    return {
+      height: `${containerHeight.value}px`,
+      overflow: 'hidden',
+    }
+  })
+
+  const carouselStyle = computed(() => {
+    if (unref(isCardType) && unref(isVertical)) {
+      return {
+        width: props.height,
+      }
+    }
+    return {}
+  })
```

Comment:
I'm not quite sure what this code does

Reply:
I think these codes are redundant

Reply:
ok, this code is really redundant and forgot to delete

Reply:
code has been updated @tolking